plugins/foreman_virt_who_configure/0.5/index.md

```diff
@@ -0,0 +1,272 @@
+
+### 3.2.1 Copy and paste
+
+You can simply copy and paste the content of the script to some file on the host where you want to deploy it. Make sure the file is only readable by root, it contains sensitive information. Also make sure, you clean the clipboard afterwards. Then simply run the script e.g. like this
+
+```sh
+bash /root/my_script
+```
+
+### 3.2.2 Downloading the script
+
+You can download the script using "Download the script" button. If you're running this on some workstation, make sure you store it at a secured location and transfer it to the target host by secure transport layer (e.g. ssh). The script contains sensitive information. After you copy it to secure location on the target host, run the script. You might need to make it executable first. Following commands illustrate the process.
+
+```sh
+scp /home/user/deploy_virt_who_config_4.sh root@satellite.example.com:/root/
+ssh root@satellite.example.com
+chmod u+x /root/deploy_virt_who_config_4.sh
+/root/deploy_virt_who_config_4.sh
+```
+
+### 3.2.3 Using the API
+
+The plugin comes with a REST API that can be used. You can use the API endpoint for downloading the script directly on the host, where you want to deploy the virt-who. The following curl command would download it. Make sure you keep it in secure place.
+
+```sh
+curl -u admin:changeme  https://foreman.example.tst/foreman_virt_who_configure/api/v2//configs/4/deploy_script.sh > /root/deploy_script.sh
+chmod u+x /root/deploy_script.sh
+/root/deploy_script.sh
+```
+
+Obviously you need to change admin/changeme credentials according to your setup. The virt-who configuration id is also part of the url, 4 in this case.
+
+### 3.2.4 Using Hammer (recommended)
+
+[Hammer CLI](https://theforeman.org/manuals/latest/#3.5.6CLI) can be used to download and/or deploy the configuration script. Note that Hammer CLI can be installed on the target host and configured to run commands against your Foreman instance. We provide a plugin for Hammer CLI that must be installed in order for this to work. 
+
+All functionality that is provided in the WebUI can be achieved using this plugin. To see how to use the Hammer CLI plugin command, see
+
+```sh
+hammer virt-who-config -h
+```
+
+#### Listing configs
+
+```sh
+$ hammer virt-who-config list
+---|--------|---------------|--------|--------------------
+ID | NAME   | INTERVAL      | STATUS | LAST REPORT AT     
+---|--------|---------------|--------|--------------------
+2  | RHV    | every 4 hours | OK     | 2021/10/12 20:13:54
+1  | VMware | every 4 hours | OK     | 2021/10/14 16:14:09
+---|--------|---------------|--------|--------------------
+```
+
+#### Deploying a configuration
+ 
+```sh
+hammer virt-who-config deploy --id 4
+```
+
+This will download the script for configuration with id 4 to a secure location, run the script and it prints the result to STDOUT. You can also use name of configuration to identify which virt-who configuration you want to deploy:
```

Comment:
```suggestion
This will download the script for configuration with id 4 to a secure location, run the script and it prints the result to STDOUT. You can also use the name of the configuration to identify which virt-who configuration you want to deploy:
```